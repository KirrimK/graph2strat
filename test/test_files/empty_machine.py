"""STATES_BEGIN
digraph stonks {

}
STATES_END"""