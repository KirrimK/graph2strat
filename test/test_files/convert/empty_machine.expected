# File generated using graph2strat by KirrimK@ENAC vunknown
from statemachine import StateMachine, State, Transition
# [graph2strat generated states and transitions]


stonks = StateMachine(None)
# [end of generated content]